first pass at implementing execute method, closes #48

nlf opened the pull request:
Also added `grunt-cli` to dev dependencies so I wouldn't have to globally install it.

vernak2539 commented on package.json:
```diff
@@ -20,6 +20,7 @@
     "chai": "~1.9.1",
     "grunt": "~0.4.5",
     "grunt-bump": "~0.0.14",
+    "grunt-cli": "^0.1.13",
```
please change this to "~0.1.13"

vernak2539 commented:
Please add test cases

aydrian commented on lib/fuel-soap.js:
```diff
@@ -328,8 +328,26 @@ FuelSoap.prototype.describe = function (type, callback) {
+			},
+      Requests: {
+        Name: type,
+        Parameters: props
```
I know this sounds kind of nitpicky, but why not params instead of props? It just seems to make better since.

gscott replied:
I believe @nlf used `props` because that's in keeping with the style of the rest of the module.

vernak2539 commented:
@aydrian @nlf @gscott I see nothing wrong with this PR minus the missing test cases. Once they are added, I will publish a new version

nlf commented:
added a test.

i had a sporadic issue with the last test in `test/specs/function-soapRequest.js` erroring with an ECONNRESET but it was super inconsistent, and happened to me before i added new tests so i didn't try to fix it.

vernak2539 commented:
published as `v1.1.0`
